lib/algorithm.ts

```diff
@@ -353,3 +354,45 @@ function hazing({ tweet, sentiment }: TweetData): Rank {
     score: 0,
   }
 }
+
+/**
+ * U.S. politics is so awesome, and everybody should love to read about it,
+ * no matter their background, interests, or country of residence.
+ * You WILL form an opinion on us!
+ */
+function usPolitics({ tweet, sentiment }: TweetData): Rank {
+  // List of hot topics, power posters, and coronavirus conspiracy
+  const topics = [
+    "🇺🇸", "american?", "patriot", "president", "states", "psyop",
+    "conservatives?", "democrat", "demonrat", "republican", "crt", "woke\\w*",
+    "wokeness", "abortion", "pro-life", "authoritarian", "drag", "ideology",
+    "trump", "biden", "obama", "aoc", "cruz", "bernie", "sanders", "mtg",
+    "ilhan", "pelosi", "lindsey", "mcconnell", "warren", "mccarthy", "kamala",
+    "romney", "tulsi", "gabbard", "rubio", "bannon", "yang", "klobuchar",
+    "crenshaw", "desantis", "buttigieg", "fauci", "feinstein", "cheney",
+    "newsom", "smollett", "beto", "schumer", "hillary", "clinton", "walsh",
+    "capitol", "house bill", "chicago",
+    "vaccine", "vaccines", "pfizer", "wuhan", "mandate", "mandates",
```

Comment:
pfizer is used internationally when discussing vaccines, boosting it here seems like a mistake

Reply:
It's all Americans: https://twitter.com/search?q=pfizer&src=typed_query